docs/src/pages/components/menus/DenseMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Divider from '@material-ui/core/Divider';
+import MenuList from '@material-ui/core/MenuList';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import Check from '@material-ui/icons/Check';
+import ArrowRight from '@material-ui/icons/ArrowRight';
+
+export default function DenseMenu() {
+  return (
+    <Paper sx={{ width: 320, maxWidth: '100%' }}>
```

Comment:
I have tried without, it seems to work. What do you think?

```suggestion
    <Paper sx={{ width: 320 }}>
```